import/marc_local.properties

```diff
@@ -72,7 +72,7 @@ author_synonyms = 101a
 publisher = custom(de.unituebingen.ub.ubtools.solrmarcMixin.IxTheoPublisher), getPublishers
 publisher_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.IxTheoPublisher), getPublishersOrUnassigned
 
-topic_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.TuelibMixin), getValuesOrUnassigned(600x:610x:611x:630x:648x:650a:650x:651x:655x)
+topic_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.TuelibMixin), getValuesOrUnassigned(600x:610x:611x:630x:648x:650a:650x:651x:655x:689abctnp)
 genre_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.TuelibMixin), getValuesOrUnassigned(600v:610v:611v:630v:648v:650v:651v:655a:655v)
 geographic_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.TuelibMixin), getValuesOrUnassigned(600z:610z:611z:630z:648z:650z:651a:651z:655z)
 era_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.TuelibMixin), getValuesOrUnassigned(600d:610y:611y:630y:648a:648y:650y:651y:655y)
```

Comment:
Zeitschlagwörter von 689 sind in Marc auch in |a umgesetzt (z.B. PPN 465147259), daher kann nicht von anderen Schlagwörter unterschieden werden. Aber vor 689 steht in einem Unterfeld der Satztyp, z.B. "Dp" für Personenschlagwort, "Dg" für Geographica, "Az" für Zeitschlagwort etc.:

689 00$Dp$0(DE-588)118580485$0(DE-576)16458496X$2gnd$aMelanchthon, Philipp$d1497-1560
689 01$Dg$0(DE-588)4008135-7$0(DE-576)208874569$2gnd$aBremen
689 02$Ds$0(DE-588)4141012-9$0(DE-576)209707763$2gnd$aAbendmahlsstreit

kann man dies als Importkriterien für facet browsing nutzen?